### ISSUE TITLE
[Request]Show Health and Armor on wrist of left hand.

### ISSUE THREAD

**Issue by tsr22** [KozGit/DOOM-3-BFG-VR#172]

**Comment by KozGit** [comment]
Some people miss this, but armor and health are currently displayed on the stat watch, the vertical bars on the very left and right of the screen indicate health and armor. ( Health on left, armor on right ).

What kind of display did you have in mind?

**Comment by tsr22** [comment]
Wow I did not know that! That is very cool and thanks for the heads up.

My idea was just something like this on the left, another watch basically:
http://imgur.com/a/xDrE8

**Comment by KozGit** [comment]
Will think about this some more - anyone else have any comments?

**Comment by jdawgzim** [comment]
I'm just wondering why the GUI that appears when I look down is not always facing forward? Hard to find health/armor. I have to look down then spin around to find it. I never know what power up or PDA I just picked up. I turned of 3rd person movement and have regular movement on. Maybe this is related?

**Comment by jdawgzim** [comment]
Having everything on one watch seems to make more sense and if the floating GUI would face forward, that'd be nice too. 

SUGGESTIONS:
1) Make stat watch a little easier to read. Health/Armor bars currently hard to read.
2) Have the floating GUI/HeadsUpDisplay reset to direction HMD is facing each time HMD is tilted downward

**Comment by KozGit** [comment]
Yeah, the health and armor bars aren't currently ideal, maybe once I get some more bugs squashed I can revisit.

You CAN change the HUD to bind to your view direction instead of body direction.  Vr Settings-> UI Options->HUD Options->Location  ( change to view. )
